Cargo.toml

```diff
@@ -54,6 +54,12 @@ pretty_env_logger = "0.4"
 tokio = { version = "1.8.0", features = ["fs", "macros", "macros", "rt-multi-thread"] }
 cool_asserts = "2.0.3"
 
+xshell = "0.2"
```

Comment:
So, CI fails because of this (dev) dependency that requires rustc 1.59 or newer while out MSRV is 1.58.

Our options:
- Don't run tests on MSRV (scary)
- Bump MSRV (not really needed otherwise, but also shouldn't hurt that much)

Reply:
What is we bump MSRV only on dev?

Reply:
We can do this (as shown by the green CI).